templates/zerver/for-companies.html

```diff
@@ -0,0 +1,35 @@
+{% extends "zerver/portico.html" %}
+
+{% block title %}
+<title>Zulip: the best group chat for companies</title>
+{% endblock %}
+
+{% block customhead %}
+{{ super() }}
+
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+{% stylesheet 'portico' %}
+{% stylesheet 'landing-page' %}
+{{ render_bundle('landing-page') }}
+
+{% endblock %}
+
+{% block portico_content %}
+
+{% include 'zerver/landing_nav.html' %}
+
+<div class="portico-landing why-page no-slide">
+    <div class="hero">
+        <h1 class="center">{% trans %}The best chat for workplaces{% endtrans %}</h1>
+        <p>Make better use of your managers' time, integrate your remote workers, and replace your low-content meetings</p>
```

Comment:
yeah. I don't think it is terrible, but I'll put it up on chat.zulip.org for ideas